hypermap/dynasty/utils.py

```diff
@@ -18,25 +20,38 @@ def get_mined_dates(text):
     return mined_dates
 
 
+def clean_text(text):
+    try:
+        text = text.decode("utf-8")
+    except UnicodeEncodeError:
+        text = text.encode("ascii", "ignore")
+    return text
```

Comment:
please add also a generic exception returning None, otherwise it would break in case we have text in different encoding than utf-8 and ascii